Gems/AtomLyIntegration/AtomBridge/Code/Source/FlyCameraInputComponent.cpp

```diff
@@ -300,6 +302,16 @@ void FlyCameraInputComponent::OnKeyboardEvent(const InputChannel& inputChannel)
     {
         m_movement.x = inputChannel.GetValue();
     }
+
+	if(channelId == InputDeviceKeyboard::Key::AlphanumericE)
+	{
+		m_movement.z = inputChannel.GetValue();
+	}
+
+	if(channelId == InputDeviceKeyboard::Key::AlphanumericQ)
+	{
+		m_movement.z = -inputChannel.GetValue();
+	}
```

Comment:
The upward and downward movement seems inconsistent from a keyboard (`Q` on the left, `E` on the right) than a joystick (`L2` on the left, `R2` on the right).

On a keyboard pressing `Q` will move the camera downwards, but on joystick pressing `L2` will move the camera upwards.

Reply:
Done.